Client/Teztez/Core/UIComponents/ActivitySelectSizeView/View/ActivitySelectSizeView.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ActivitySelectSizeView.swift
+//  Teztez
+//
+//  Created by Adlet on 5/2/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import UIKit
+
+protocol ActivitySelectSizeViewDelegate: class {
```

Comment:
We don't need this view. You can use TextInputView, because SelectSizeView do the same job and its 100% copy of TextInputView. We will change the name of TextInputView in next time, don't worry